src/main/java/org/opensrp/service/EventService.java

```diff
@@ -30,11 +33,20 @@
 	private final EventsRepository allEvents;
 	
 	private ClientService clientService;
+
+	private TaskGenerator taskGenerator;
+
+	private PlanRepository planRepository;
+
+	@Value("#{opensrp['plan.evaluation.enabled']}")
```

Comment:
Add a default of false

Reply:
Changed its data type to primitive boolean